### PR TITLE
Auto complete command implemented

### LINKED ISSUE #67: Add auto-complete command feature

### PR DESCRIPTION
**Pull request by AdityaA1998**
close #67, #74, #75

### CODE REVIEW

File: src/main/java/seedu/progresschecker/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -37,6 +37,14 @@
 
     public static final String COMMAND_WORD = "edit";
     public static final String COMMAND_ALIAS = "ed";
+    public static final String COMMAND_FORMAT = COMMAND_WORD + " "
+                        + "INDEX "
```
Comment (kohyeeru) [inline src/main/java/seedu/progresschecker/logic/commands/EditCommand.java]: Is the indentation off here or..?

File: src/main/java/seedu/progresschecker/logic/commands/SortCommand.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
  */
 public class SortCommand extends UndoableCommand {
     public static final String COMMAND_WORD = "sort";
+    public static final String COMMAND_ALIAS = "so";
```
Comment (kohyeeru) [inline src/main/java/seedu/progresschecker/logic/commands/SortCommand.java:10]: Might need to reconsider formatting of all alias for consistency? (Not asking for changes in this PR however)

File: src/main/java/seedu/progresschecker/ui/CommandBox.java [hunk 3]
```diff
@@ -171,4 +179,22 @@ private void setStyleToIndicateCommandFailure() {
 
         styleClass.add(ERROR_STYLE_CLASS);
     }
+
+    /**
+     * @param text
```
Comment (kohyeeru) [inline src/main/java/seedu/progresschecker/ui/CommandBox.java]: Any description for what this param is?

**Review by kohyeeru — CHANGES_REQUESTED** [review]
Review consist of the mixture of changes requested and comments that are for your future consideration only.

File: src/main/java/seedu/progresschecker/ui/CommandBox.java (hunk 3 above)
Comment (AdityaA1998) [reply src/main/java/seedu/progresschecker/ui/CommandBox.java]: forgot to add

**Review by kohyeeru — APPROVED** [review]

**Pull request merged**